### ISSUE TITLE
An error is occurred while installing the dotfiles on Fedora

### ISSUE THREAD

**Issue by TsutomuNakamura** [TsutomuNakamura/dotfiles#45]
An error is occurred while installing the dotfiles on Fedora due to lack of clang and clang-devel. The error messages are like below.

```
CMake Error at ycm/CMakeLists.txt:330 (message):
  Using Clang completer, but no libclang found.  Try setting
  EXTERNAL_LIBCLANG_PATH or revise your configuration
```

**Comment by TsutomuNakamura** [comment]
ubuntu also has an error similar to this.
Package 'build-essential' will be necessary.

**Comment by TsutomuNakamura** [comment]
I decided not to support CentOS so far. Because some important packages like python3 etc are not supported officially and yum can't install build-essentials like dnf. Alternativelly, yum can install Development tools but the change for them will make a lot of cost to test them.

**Comment by TsutomuNakamura** [comment]
Closed by a pull request #46.
